docs/painless/painless-contexts.asciidoc

```diff
@@ -0,0 +1,58 @@
+[[painless-contexts]]
+== Painless Contexts
+
```

Comment:
This is a style nit (and not sure I've caught it before), but we've agreed on Sentence style capitalization for headings. It's far from consistent across the docs, but we're trying to move in that direction, especially with new content.

Reply:
Fixed.